webapp/src/app/shared/phaser/helper.ts

```diff
@@ -100,9 +109,13 @@ export class Helper {
 		if (scene.textures.exists(key)) {
 			return true;
 		}
-		
+		const truePath = await Helper.modloaderService.getResourcePath(key);
 		return new Promise(res => {
-			scene.load.image(key, Globals.URL + key);
+			if (Globals.isElectron) {
```

Comment:
`Helper.modloaderService.getResourcePath(key);` should handle differences between electron and web build